src/components/Select/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ChangeHandler } from 'react-hook-form';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import {
+  MenuItem,
+  Select as MUISelect,
+  SelectProps as MUISelectProps,
+} from '@mui/material';
+
+export interface IMenuItem {
+  itemValue: number;
+  itemLabel: string;
+}
+export interface SelectProps extends MUISelectProps {
+  items: IMenuItem[];
+  className?: string;
+  handleChange?: ChangeHandler;
+  defaultLabel?: string;
+}
+
+export const Select = ({
+  items,
+  handleChange,
+  defaultLabel,
+  sx,
+  ...others
+}: SelectProps) => {
+  const menuItems = () =>
+    items.map(item => {
+      const menuLabel = item.itemLabel;
+      const menuValue = item.itemValue;
+      return (
+        <MenuItem key={menuLabel} value={menuValue}>
+          {menuLabel}
+        </MenuItem>
+      );
+    });
+
+  return (
+    <MUISelect
+      variant="standard"
+      id="select"
+      value={defaultLabel}
+      onChange={handleChange}
+      IconComponent={ExpandMoreIcon}
+      sx={{
+        ...sx,
```

Comment:
...sx b phải cho nó xuống cuối chứ

Reply:
FE hay để xuống cuối à. t làm BE toàn cho lên đầu